### PR TITLE
Blog on MP 3.2 compatibility with Quarkus

### PR DESCRIPTION
**Pull request by kenfinnigan**

### CODE REVIEW

**Comment by kenfinnigan** [comment]
NOTE: Don't merge this until we know 1.1 will be released by Wednesday. I somewhat guessed on the date as I will be on PTO

File: _posts/2019-12-18-quarkus-eclipse-microprofile-3-2-compatible.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+but also fixes for dependent libraries such as https://smallrye.io/[SmallRye] and https://resteasy.github.io/[RESTEasy].
+
+With Quarkus we’re seeking the best developer experience for cloud-native development,
+and sometimes that means making tough decisions.
+With ArC we sought to create an opinionated CDI implementation optimized for the cloud and containers, and in particular, Kubernetes.
+ArC isn’t a full CDI implementation because it’s engineered to take full advantage of the build time optimization within Quarkus,
+enabling supersonic runtime startup and subatomic memory utilization.
+The slight drawback is that not all CDI features are available,
+but features that developers generally use on a day-to-day basis are present.
+Check out https://quarkus.io/guides/cdi-reference for the details.
```
Comment (gsmet) [inline _posts/2019-12-18-quarkus-eclipse-microprofile-3-2-compatible.adoc]: It's hard to see how this whole paragraph articulates with MP compatibility.

Comment (kenfinnigan) [reply _posts/2019-12-18-quarkus-eclipse-microprofile-3-2-compatible.adoc]: Not sure what you mean.

Comment (gsmet) [reply _posts/2019-12-18-quarkus-eclipse-microprofile-3-2-compatible.adoc]: You're talking about MP compatibility then you're jumping to ArC and CDI without any obvious link between them.

Comment (kenfinnigan) [reply _posts/2019-12-18-quarkus-eclipse-microprofile-3-2-compatible.adoc]: I'm trying not to draw too much attention while acknowledging ArC wouldn't pass CDI TCK

**Comment by kenfinnigan** [comment]
Let me know if this can be merged, would be good to have it out next week.

Any updates @jclingan?

File: _posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: Quarkus is Eclipse MicroProfile 3.2 compatible!
+date: 2020-01-07
+tags: quarkus microprofile
+author: kenfinnigan
+---
+
+With the recent announcement that Quarkus 1.1 is now available,
+and after a tremendous effort from all the contributors and community,
+we’re also excited to announce that Quarkus 1.1 is Eclipse MicroProfile 3.2 compatible!
+https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-3.2[MicroProfile 3.2] was released this week,
```
Comment (jmartisk) [inline _posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc]: MP 3.2 was released back in November AFAIK (https://github.com/eclipse/microprofile/releases/tag/3.2) so this "last week" reference doesn't look right to me in a post dated 2020-01-07

File: _posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+but also fixes for dependent libraries such as https://smallrye.io/[SmallRye] and https://resteasy.github.io/[RESTEasy].
+
+With Quarkus we’re seeking the best developer experience for cloud-native development,
+and sometimes that means making tough decisions.
+With ArC we sought to create an opinionated CDI implementation optimized for the cloud and containers, and in particular, Kubernetes.
+ArC isn’t a full CDI implementation because it’s engineered to take full advantage of the build time optimization within Quarkus,
+enabling supersonic runtime startup and subatomic memory utilization.
+The slight drawback is that not all CDI features are available,
+but features that developers generally use on a day-to-day basis are present.
+Check out https://quarkus.io/guides/cdi-reference for the details.
```
Comment (jmartisk) [inline _posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc]: Other hyperlinks contain a label (in square brackets), perhaps this one could too for consistency?

**Comment by kenfinnigan** [comment]
Thanks @jmartisk, those should be fixed now

**Comment by kenfinnigan** [comment]
@emmanuelbernard @n1hility can I merge this so I can post it?

**Comment by emmanuelbernard** [comment]
We discussed who would review and @n1hility had more value than me so he 
got the straw ;)

**Comment by kenfinnigan** [comment]
Is there a desire to have this published?

**Comment by kenfinnigan** [comment]
I've reached a point where I'm not even sure there's much point in publishing this as WildFly has already announced it's MP 3.2 compatible!

Thoughts @jclingan, @emmanuelbernard, @n1hility?

**Comment by kenfinnigan** [comment]
Thanks @n1hility 

I might do a quick change to today instead of tomorrow, and then push if that's ok

**Pull request merged**